src/pages/changes.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import Layout from '../components/layout';
+import { StaticQuery, graphql } from 'gatsby';
+import { Document } from '@contentful/rich-text-types';
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer';
+
+import '../styles/changes.scss';
+
+type ChangelogEntry = {
+  title: string,
+  date: string, // e.g. "2019-10-01"
+  body: {
+    json: Document,
+  },
+  node_locale: string,
+};
```

Comment:
Ideally this should come from the GraphQL infrastructure itself, since this is ultimately defined in a GraphQL schema, which is what happens on the tenant platform... but everything seems untyped here so I manually wrote the typing.

Reply:
yeah... I've been doing the same manual typings for some of the smaller blocks we have (especially in the Learning Center) but have left larger content chunks untyped. There is a Clubhouse card for it though!